src/navigation/routeKeys.ts

```diff
@@ -1,14 +1,14 @@
 export const dashboard = 'Dashboard' as const;
-export const tips = 'Tips' as const;
-export const tipDetail = 'TipDetail' as const;
-export const submitTip = 'SubmitTipScreen' as const;
-export const registrarList = 'RegistrarList' as const;
-export const motionDetail = 'MotionDetail' as const;
-export const notificationSettingsScreen = 'NotificationSettingsScreen' as const;
+export const tipsScreen = 'Tips' as const;
```

Comment:
NIT: Why only adding "screen" to this one? I think we should either drop or add "Screen" to all of them

Reply:
I added `Screen` to all 👍